packages/issuer/lib/VcIssuer.ts

```diff
@@ -100,10 +106,19 @@ export class VcIssuer {
         throw new Error(KID_JWK_X5C_ERROR)
       }
 
-      const { aud, iat, nonce } = payload
-      // I couldn't find a way to get the client_id, neither the grant type at this point, so I will not validate iss since it's optional
+      const { iss, aud, iat, nonce } = payload
       // iss: OPTIONAL (string). The value of this claim MUST be the client_id of the client making the credential request.
       // This claim MUST be omitted if the Access Token authorizing the issuance call was obtained from a Pre-Authorized Code Flow through anonymous access to the Token Endpoint.
+      // TODO We need an introspection endpoint in case the AS and RS are separated
```

Comment:
Please remove this comment, because we would use access tokens, meaning no overhead of calling the AS from the RS is necessary to begin with. This comment will confuse future developers into the wrong direction

Reply:
Fixed!